### ISSUE TITLE
tslint should show errors/warnings for the whole project not only for loaded buffers

### ISSUE THREAD

**Issue by zoechi** [neoclide/coc-tslint#6]
Perhaps it's a configuration issue on my side or not in the scope of the project,
but that's what I expected and it doesn't do it for me.

```
2019-12-12T13:17:09.643 INFO (pid:16357) [services] - registered service "languageserver.elmLS"
2019-12-12T13:17:09.684 INFO (pid:16357) [services] - registered service "highlight"
2019-12-12T13:17:10.100 INFO (pid:16357) [services] - registered service "tslint"
2019-12-12T13:17:10.101 INFO (pid:16357) [services] - tslint server state change: stopped => starting
2019-12-12T13:17:10.145 INFO (pid:16357) [services] - registered service "tsserver"
2019-12-12T13:17:10.251 INFO (pid:16357) [plugin] - coc 0.0.74-d83536aa4e initialized with node: v13.3.0
2019-12-12T13:17:10.262 INFO (pid:16357) [language-client-index] - angular started with 16405
2019-12-12T13:17:10.276 INFO (pid:16357) [language-client-index] - highlight started with 16412
2019-12-12T13:17:10.287 INFO (pid:16357) [language-client-index] - tslint started with 16418
2019-12-12T13:17:10.322 INFO (pid:16357) [services] - service tsserver started
2019-12-12T13:17:10.361 INFO (pid:16357) [services] - highlight server state change: starting => running
2019-12-12T13:17:10.695 INFO (pid:16357) [services] - tslint server state change: starting => running
2019-12-12T13:17:10.698 INFO (pid:16357) [services] - service tslint started
```

**Comment by chemzqm** [comment]
No, you have to execute project lint command

**Comment by zoechi** [comment]
Do you mean the `tsline.lintProject` action?
I tried that but didn't change anything.

**Comment by chemzqm** [comment]
It all diagnostic locations to your quickfix list

**Comment by zoechi** [comment]
The list is still empty after that. Only when I run `tslint` in a terminal to look up which files have issues and open such a file in vim, then diagnostics shows entries that I can work with.

**Comment by zoechi** [comment]
tsserver.openTsServerLog shows
```
Info 38   [13:48:53.172] Failed to load module 'typescript-tslint-plugin': Error: Could not resolve JS module 'typescript-tslint-plugin'
```
I fixed that but after restart it's still the same.

Another exception is shown
```
-----------------------------------------------

Info 204  [13:55:45.491] event:
    {"seq":0,"type":"event","event":"requestCompleted","body":{"request_seq":5}}
Info 205  [13:55:45.677] request:
    {"seq":6,"type":"request","command":"geterr","arguments":{"delay":0,"files":["/home/zoechi/Customers/Cons/ClientApp/src/modules/staff/scale-search/scale-search.component.ts"]}}
Perf 206  [13:55:45.677] 6::geterr: async elapsed time (in milliseconds) 0.3135
Err 207   [13:55:45.679] Exception on executing command delayed processing of request 6:

    Could not find sourceFile: '/home/zoechi/Customers/Cons/ClientApp/src/modules/staff/scale-search/scale-search.component.ts' in [].

    Error: Could not find sourceFile: '/home/zoechi/Customers/Cons/ClientApp/src/modules/staff/scale-search/scale-search.component.ts' in [].
        at getValidSourceFile (/home/zoechi/Customers/Cons/ClientApp/node_modules/typescript/lib/tsserver.js:120714:23)
        at Proxy.getSyntacticDiagnostics (/home/zoechi/Customers/Cons/ClientApp/node_modules/typescript/lib/tsserver.js:120908:52)
        at IOSession.Session.syntacticCheck (/home/zoechi/Customers/Cons/ClientApp/node_modules/typescript/lib/tsserver.js:129298:87)
        at checkOne (/home/zoechi/Customers/Cons/ClientApp/node_modules/typescript/lib/tsserver.js:129330:27)
        at MultistepOperation.executeAction (/home/zoechi/Customers/Cons/ClientApp/node_modules/typescript/lib/tsserver.js:128550:25)
        at /home/zoechi/Customers/Cons/ClientApp/node_modules/typescript/lib/tsserver.js:128540:100
        at IOSession.Session.executeWithRequestId (/home/zoechi/Customers/Cons/ClientApp/node_modules/typescript/lib/tsserver.js:130457:28)
        at Object.executeWithRequestId (/home/zoechi/Customers/Cons/ClientApp/node_modules/typescript/lib/tsserver.js:129106:87)
        at Timeout._onTimeout (/home/zoechi/Customers/Cons/ClientApp/node_modules/typescript/lib/tsserver.js:128540:41)
        at listOnTimeout (internal/timers.js:536:17)
        at processTimers (internal/timers.js:480:7)

Projects::

Project '/home/zoechi/Customers/Cons/ClientApp/tsconfig.json' (Configured) 0
	Files (0)

-----------------------------------------------
```
I don't know what this is about.

**Comment by chemzqm** [comment]
Looks like bad tsconfig.json

**Comment by zoechi** [comment]
Could be https://github.com/theia-ide/typescript-language-server/issues/109

**Comment by zoechi** [comment]
What about coc-eslint? Should that show errors/warnings for the whole project
or only for loaded files?
There is no related `esline.lintProject` action.

**Comment by chemzqm** [comment]
You have to use `:CocCommand tslint.lintProject` for lint your whole project.

**Comment by zoechi** [comment]
Ok, thanks!
